content/en/docs/release-oversight/improving-ci-signal.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Improving CI Signal"
+description: An in-depth look at how job aggregation helps detect payload regressions
+---
+
```

Comment:
The content below is based on a document from @deads2k.  I changed some of the verb tense now that some of this is in place.  I also removed several pages from the middle of the document that were primarily focused on building the case that this approach is needed.